src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -19,6 +19,16 @@
 -->
 
 ## Upcoming Release
+* Introduced support for creating accounts with continuous mode backup policy.
+* Introduced support for Point in time restore for accounts with continuous mode backup policy.
```

Comment:
Add the information that this is a module exposing preview features. Also mention the fact that throughput migration cmdlets (and analytical ttl for mongo collection) is removed

Reply:
Added the info